packages/http/src/validator/validators/params.ts

```diff
@@ -19,60 +19,58 @@ export type Deps<Target> = {
+      .filter(spec => spec.deprecated && target[spec.name])
+      .map<IPrismDiagnostic>(spec => ({
+        path: [prefix, spec.name],
+        code: 'deprecated',
+        message: `${upperFirst(prefix)} param ${spec.name} is deprecated`,
+        severity: DiagnosticSeverity.Warning,
+      }));
 
-  return pipe(
-    NEA.fromArray(specs),
-    O.map(specs => {
-      const schema = schemaCache.get(specs) ?? createJsonSchemaFromParams(specs);
-      if (!schemaCache.has(specs)) {
-        schemaCache.set(specs, schema);
-      }
+    return pipe(
+      NEA.fromArray(specs),
+      O.map(specs => {
+        const schema = schemaCache.get(specs) ?? createJsonSchemaFromParams(specs);
+        if (!schemaCache.has(specs)) {
+          schemaCache.set(specs, schema);
+        }
 
-      const parameterValues = pickBy(
-        mapValues(
-          keyBy(specs, s => s.name.toLowerCase()),
-          el => {
-            const resolvedStyle = el.style || defaultStyle;
-            const deserializer = deserializers[resolvedStyle];
+        const parameterValues = pickBy(
+          mapValues(
+            keyBy(specs, s => s.name.toLowerCase()),
+            el => {
+              const resolvedStyle = el.style || defaultStyle;
+              const deserializer = deserializers[resolvedStyle];
 
-            return deserializer
-              ? deserializer(
-                  el.name.toLowerCase(),
-                  // @ts-ignore
-                  mapKeys(target, (_value: unknown, key: string) => key.toLowerCase()),
-                  schema.properties && (schema.properties[el.name.toLowerCase()] as JSONSchema4),
-                  el.explode || false
-                )
-              : undefined;
-          }
-        )
-      );
-      return { parameterValues, schema };
-    }),
-    O.chain(({ parameterValues, schema }) => validateAgainstSchema(parameterValues, schema, true, prefix, bundle)),
-    O.map(schemaDiagnostic => NEA.concat(schemaDiagnostic, deprecatedWarnings)),
-    O.alt(() => NEA.fromArray(deprecatedWarnings)),
-    E.fromOption(() => target),
-    E.swap
-  );
-};
+              return deserializer
+                ? deserializer(
+                    el.name.toLowerCase(),
+                    // @ts-ignore
+                    mapKeys(target, (_value: unknown, key: string) => key.toLowerCase()),
+                    schema.properties && (schema.properties[el.name.toLowerCase()] as JSONSchema4),
+                    el.explode || false
+                  )
+                : undefined;
+            }
+          )
+        );
+        return { parameterValues, schema };
+      }),
+      O.chain(({ parameterValues, schema }) => validateAgainstSchema(parameterValues, schema, true, prefix, bundle)),
+      O.map(schemaDiagnostic => NEA.concat(schemaDiagnostic, deprecatedWarnings)),
+      O.alt(() => NEA.fromArray(deprecatedWarnings)),
+      E.fromOption(() => target),
+      E.swap
+    );
+  };
```

Comment:
Is this just formatting changes?  (I tried to read through this, but couldn't pick out any code changes.)